src/rws_interface.cpp

```diff
@@ -818,6 +818,19 @@ bool RWSInterface::setMotorsOff()
   return rws_client_.setMotorsOff().success;
 }
 
+bool RWSInterface::setSpeedRatio(unsigned int ratio)
+{
+  if(ratio > 100)
+  {
+    ratio = 100;
+  }
+
+  std::stringstream ss;
+  ss << ratio;
```

Comment:
I know this "should not fail", but it would perhaps be nice to add some checking here, just in case.

Similar to all the previous PRs against `abb_librws`.

Reply:
Added a check in 4a99d0a.